src/routes/common/logging/logging.service.spec.ts

```diff
@@ -6,37 +6,36 @@ const mockClsService = {
   getId: jest.fn(() => '123-456'),
 } as unknown as ClsService;
 
-jest.mock('winston');
+const mockLogger = {
+  info: jest.fn(),
+  error: jest.fn(),
+  warn: jest.fn(),
+  debug: jest.fn(),
+} as unknown as winston.Logger;
 
 describe('RequestScopedLoggingService', () => {
-  const infoMock = jest.fn();
-  const debugMock = jest.fn();
-  const errorMock = jest.fn();
-  const warnMock = jest.fn();
   let loggingService: RequestScopedLoggingService;
 
   beforeAll(() => {
     jest.useFakeTimers();
     jest.setSystemTime(new Date('2023-01-01'));
-    jest.spyOn(winston, 'info').mockImplementation(infoMock);
-    jest.spyOn(winston, 'debug').mockImplementation(debugMock);
-    jest.spyOn(winston, 'error').mockImplementation(errorMock);
-    jest.spyOn(winston, 'warn').mockImplementation(warnMock);
-
-    loggingService = new RequestScopedLoggingService(mockClsService);
   });
 
   beforeEach(() => {
     jest.clearAllMocks();
+    loggingService = new RequestScopedLoggingService(
+      mockLogger,
+      mockClsService,
+    );
```

Comment:
Moved the instance creation from `beforeAll` to `beforeEach` to avoid any state sharing between the tests. At the moment no state is stored in this class but if that changes we might start getting flaky tests (as the shared state can affect the result of the assertions).